project/maps2params/python/get_maps_to_params_spectra.py

```diff
@@ -102,11 +107,16 @@
                 alms_beamed[1] += fglms[2*fcount+1]
 
             alms_beamed = maps_to_params_utils.multiply_alms(alms_beamed, bl, ncomp)
+            
+            #here we calibrate the T, E alms and rotate the E, B alms of each frequency, for the LAT experiment
+            if exp == "LAT":
+                cal_alms = maps_to_params_utils.calibrate_alm(alms_beamed, exp, freq, **nuis_params)
+                sys_alms = maps_to_params_utils.rotate_alm_polang(cal_alms, exp, freq, **nuis_params)
```

Comment:
I think here you need an `else` statement to avoid bug when someone will pass another experiment than `LAT` (we never know what users can do)
```python
else:
  sys_alms = alms_beamed
```